jaseci_core/jaseci/graph/edge.py

```diff
@@ -123,6 +123,18 @@ def connects(self, source=None, target=None, ignore_direction=False):
                 return False
         return True
 
+    def set_context(self, ctx, arch=None):
```

Comment:
@marsninja I copied this function over from the node class. I am not 100% sure what the `arch` parameter is used for though...

Reply:
perfect, arch is for updated architypes if the edge itself is old.